### DISCUSSION TITLE
Bundler could not find compatible versions for gem "actionmailer":

### DISCUSSION THREAD

**Discussion by jessicana** [solidusio/solidus#4364] [Troubleshooting] [answered]
I am trying to install Solidus and run across this error. How can I resolve this conflict?

![image](https://user-images.githubusercontent.com/7415824/165672819-0aa7ba1a-8678-4487-bfe2-6333558cf74b.png)

**Comment by waiting-for-dev** [comment, marked as answer]
Hey @jessicana, which Solidus version are you trying to install?

**Reply by jessicana** [reply]
@waiting-for-dev v2.11.15

**Reply by waiting-for-dev** [reply]
Hey @jessicana. That version is not compatible with Rails 7. It supports [Rails 5.2 to Rails 6.1](https://github.com/solidusio/solidus/blob/86985ada665f0b19686a5715bf31c3912a2f184b/core/solidus_core.gemspec#L28). For Rails 7 support you need to use the `master` branch, although we're going to release v3.2 very soon.
